crates/biome_cli/src/execute/mod.rs

```diff
@@ -117,7 +117,7 @@ impl Execution {
         Self {
             report_mode: ReportMode::default(),
             traversal_mode: mode,
-            max_diagnostics: MAXIMUM_DISPLAYABLE_DIAGNOSTICS,
+            max_diagnostics: 20,
```

Comment:
Is this an error of switching of `200` to `20`?

Reply:
The CLI default is 20 :) 

https://github.com/biomejs/biome/blob/c98a030f6eb1badb4790cfbca5d4d238efd94f2c/crates/biome_cli/src/cli_options.rs#L30